tests/conn_can/README.md

```diff
@@ -22,21 +22,21 @@ sudo apt-get install libsocketcan-dev
 
 Alternatively, you can compile from source:
 ```
-wget http://www.pengutronix.de/software/libsocketcan/download/libsocketcan-0.0.10.tar.bz2
+wget http://www.pengutronix.de/software/libsocketcan/download/libsocketcan-0.0.11.tar.bz2
 
-$ sudo tar xvjf libsocketcan-0.0.10.tar.bz2
+$ tar xvjf libsocketcan-0.0.10.tar.bz2
```

Comment:
Well now you also need to change the version number here ;)

Reply:
You're right! Thank you for the hint.